docs/docs/cmd/graph/subscription/subscription-add.mdx

```diff
@@ -67,6 +71,24 @@ Create a subscription using the maximum allowed expiration for Group resources
 m365 graph subscription add --resource groups --changeTypes "updated" --notificationUrl "https://webhook.azurewebsites.net/api/send/myNotifyClient"
 ```
 
+Create a subscription for Event Hub location using by Key Vault
```

Comment:
`using by Key Vault` sounds odd to me, let's rephrase it a bit. (same for the next description.)

Reply:
fixed